tokio/src/process/mod.rs

```diff
@@ -346,7 +346,7 @@ impl Command {
     /// whether it should be interpreted relative to the parent's working
     /// directory or relative to `current_dir`. The behavior in this case is
     /// platform specific and unstable, and it's recommended to use
-    /// [`canonicalize`] to get an absolute program path instead.
+    /// `canonicalize` to get an absolute program path instead.
```

Comment:
If there's no `tokio` equivalent of this function we should link to the `std` one

Reply:
I didn't look into whether or not we needed one yet.

Reply:
I believe we do need a version.

Reply:
Ok, I added it and fixed the link.